src/_components/Category/CategoryContainer.jsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import CategoryComponent from '../../components/Category/AddCategoryComponent';
+import { createCategory } from '../../_actions/category.actions';
+import { ToastMessage } from '../../_utils/ToastMessage';
+import resetToastMessageContent from '../../_actions/resetToastMessage.actions';
+
+
+class CategoryContainer extends React.Component {
+  constructor(props) {
```

Comment:
A constructor is not necessary when there's no state being initialized with a prop value. Initialize the state as a class variable. I.e.
```
class CategoryContainer extends React.Component {
  state = {
    categoryName: '',
    saveButtonState: false
  }
  ...
  ...
  render() {...}
}
```